projects/flourish-web-sdk-angular/src/lib/enums/environment.enum.ts

```diff
@@ -1,5 +1,20 @@
-enum Environment {
+export enum Environment {
```

Comment:
I added maximum possibilities to be "stupid client proof", what you guys think?

Reply:
I dislike. It's your enum, the customer should translate them environment to your enum and we should have only one option per environment.